### DISCUSSION TITLE
Avoid autocomplete html structure when generating template

### DISCUSSION THREAD

**Discussion by saiballo** [11ty/eleventy#2984] [Uncategorized]
I'm using njk syntax with eleventy to generate some snippets of html code for another project.

if I compile njk template in src folder like this:

``<p>Hello!</p>
``

It will compile with more html tags:

`` 
&lt;!doctype html>
&lt;html>
&lt;head>&lt;/head>
&lt;body>
<p>Hello</p>
&lt;/body>&lt;/html>``

How can tell to eleventy to avoid adding full html structure around my page code?

**Comment by pdehaan** [comment]
Works for me&hellip; https://github.com/pdehaan/11ty-2984/
Might be some other plugin that's adding that? 

### src/index.njk <small>[[1]](https://github.com/pdehaan/11ty-2984/blob/main/src/index.njk)</small>
```html
Hello!
```

### www/index.html <small>[[2]](https://github.com/pdehaan/11ty-2984/blob/main/www/index.html)</small>
```html
Hello!
```

**Reply by saiballo** [reply]
Mmm... You are right!  I Need to check my plugin list. Thank you for your suggestion

**Reply by pdehaan** [reply]
Also, if you have a global/directory `layout` defined which adds some extra UI, that might cause issues.

**Comment by saiballo** [comment]
The problem was "eleventy-plugin-external-links" that adds some extra code. Now I disabled it and every goes right

https://www.npmjs.com/package/@aloskutov/eleventy-plugin-external-links
